### PR TITLE
Updates styleguide to v1

### PR DESCRIPTION
**Pull request by cindywu**
**v1**
- [x] all headers use [Roboto](https://fonts.google.com/specimen/Roboto) (updates h1 to serif font from sans-serif font)
- [x] removes Sumana from ```$serifFontFamily``` defaulting to Noto Serif, Georgia, Times
- [x] adds [Noto Serif](https://fonts.google.com/specimen/Noto+Serif) as our serif font, used for ```p``` content (required to support latin and greek characters)
- [x] define styles for h1, h2, h3, h4, p

**v2**
- [ ] define button styles
- [ ] define black color for text
- [ ] define link styles

**vn**
replace h1 with serif font that feels like a heavier and wider cousin of Times, inspired by Tiempos Headline

### CODE REVIEW

File: app/assets/stylesheets/core/_type.scss [hunk 1]
```diff
@@ -35,11 +37,10 @@ h1, h2, h3, h4, h5, h6 {
   text-rendering: optimizelegibility; // Fix the character spacing for headings
 }
 
-h1 { font-size: 29px; }
-h2 { font-size: 24px; }
-h3 { font-size: 21px; }
-h4 { font-size: 17px; }
-h5 { font-size: 17px; }
+h1 { font-size: 40px; line-height: 47px; }
```
Comment (dennyluan) [inline app/assets/stylesheets/core/_type.scss]: you might want to install a whitespace remover on your editor, or if you're using sublime add `trim_trailing_white_space_on_save: true` to your user preferences.

File: app/assets/stylesheets/core/_variables.scss [hunk 2]
```diff
@@ -65,10 +65,10 @@ $twitter:         #00AAF1;
 $reddit:          #FF3A00;
 
 // Type variables
-$baseFontSize: 				16px;
+$baseFontSize: 				14px;
 $baseLineHeight:      20px;
 $baseFontFamily:      'Roboto', Helvetica, Arial, sans-serif;
-$serifFontFamily:     'Gelasio', Georgia, Times, serif;
+$serifFontFamily:     'Noto Serif', Georgia, Times, serif;
```
Comment (dennyluan) [inline app/assets/stylesheets/core/_variables.scss]: If we're going to change the font then you also need to change/update the google font import in `layouts/application` and remove the Gelasio one.

File: app/assets/stylesheets/core/_type.scss (hunk 1 above)
Comment (cindywu) [reply app/assets/stylesheets/core/_type.scss]: ok! i updated ```trim_trailing_white_space_on_save: true``` on my sublime.

however, it seems our preference is adding a space after ```;``` before ```}```. is this true?

```h1 { font-size: 40px; line-height: 47px; }
h2 { font-size: 24px; line-height: 28px; }
h3 { font-size: 20px; line-height: 23px; }
h4 { font-size: 16px; line-height: 19px; }```

File: app/assets/stylesheets/core/_variables.scss (hunk 2 above)
Comment (cindywu) [reply app/assets/stylesheets/core/_variables.scss]: updated google font import https://fonts.google.com/?query=robot&selection.family=Noto+Serif:400,400i,700|PT+Mono|Roboto:400,500,700,700i&selection.subset=cyrillic,cyrillic-ext,greek,greek-ext

File: app/assets/stylesheets/core/_type.scss (hunk 1 above)
Comment (cindywu) [reply app/assets/stylesheets/core/_type.scss]: @dennyluan said what is important is that our formatting is are consistent

**Comment by dennyluan** [comment]
LGTM

**Pull request merged**